overlord/configstate/configcore/homedirs.go

```diff
@@ -73,6 +74,30 @@ func init() {
 	supportedConfigurations["core.homedirs"] = true
 }
 
+func configureAppArmorAndReload(homedirs []string, opts *fsOnlyContext) error {
```

Comment:
this maybe should be called configureHomedirsInAppArmorAndReload ?

Reply:
Yeah that's more clear, i'll get that updated